public/js/user.js

```diff
@@ -340,7 +373,8 @@ function updateMarker(id) {
       contentType: "application/json",
     })
         .done(function(response) {
-          sites[id].siteData = response; 
+          sites[id].siteData = response;
+          sites[id].siteData.image = bufferToString(sites[id].siteData.image);
```

Comment:
This didn't appear to be fully working for me (testing in Chrome). I had to manually reload the page before the marker's bubble showed the image.